src/analysis/retail/warlock/destruction/modules/talents/Eradication.tsx

```diff
@@ -10,9 +10,10 @@ import { calculateEffectiveDamage } from 'parser/core/EventCalculateLib';
 import Events, { CastEvent, DamageEvent } from 'parser/core/Events';
 import { When, NumberThreshold, ThresholdStyle } from 'parser/core/ParseResults';
 import Enemies from 'parser/shared/modules/Enemies';
-import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
 import Statistic from 'parser/ui/Statistic';
 import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import TalentSpellText from 'parser/ui/TalentSpellText';
+
 
 const MAX_TRAVEL_TIME = 3000; // Chaos Bolt being the slowest, takes around 2 seconds to land from max range, added a second to account for maybe target movement?
 const ERADICATION_DAMAGE_BONUS = 0.1;
```

Comment:
This talent is a 2-point talent for 5% and 10%, this constant should be defined as .05 and then have a var defined as `ERADICATION_DAMAGE_BONUS * this.selectedCombatant.getTalentRank(TALENTS.ERADICATION_TALENT)`